scripts/xtensa-build-zephyr.sh

```diff
@@ -249,10 +251,10 @@ build_platforms()
 			    # CMAKE_ARGS this _not_ does force CMake to re-run and passing
 			    # a different board by mistake is very nicely caught by west.
 			    set -x
-			    west build --build-dir "$bdir" --board "$PLAT_CONFIG"
+			    west ${BUILD_VERBOSE} build --build-dir "$bdir" --board "$PLAT_CONFIG"
 			else
 			    set -x
-			    west build --build-dir "$bdir" --board "$PLAT_CONFIG" \
+			    west ${BUILD_VERBOSE} build --build-dir "$bdir" --board "$PLAT_CONFIG" \
```

Comment:
> Double quote to prevent globbing and word splitting. [SC2086]

You can both avoid the warning _and_ make the option more flexible with one layer of indirection

```suggestion
			local verbose
			[ -z "$BUILD_VERBOSE" ] || verbose=-v
                         ...
			west ${verbose} build --build-dir "$bdir" --board "$PLAT_CONFIG" \
```

Reply:
Thanks, fixed!